### PR TITLE
add accelerate sagemaker examples

### PR DESCRIPTION
**Pull request by pacman100**
# What does this PR do?
1. End-to-end examples on how to use AWS SageMaker integration of Accelerate

### CODE REVIEW

**Review by philschmid — APPROVED** [review]
Awesome 😍 Thank you for adding those!

File: sagemaker/22_accelerate_sagemaker_examples/README.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
+# accelerate-aws-sagemaker
+Examples showcasing AWS SageMaker integration of 🤗 Accelerate. Just give the `accelerate config` and do `accelerate launch` 🚀. As simple as that!
```
Comment (sgugger) [inline sagemaker/22_accelerate_sagemaker_examples/README.md:2]:
```suggestion
Examples showcasing AWS SageMaker integration of 🤗 Accelerate. Just give the `accelerate config` and do `accelerate launch` 🚀. It's as simple as that!
```

File: sagemaker/22_accelerate_sagemaker_examples/README.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+1. Set up the accelerate config by running `accelerate config --config_file accelerate_config.yaml` and answer the SageMaker questions and set it up.
```
Comment (sgugger) [inline sagemaker/22_accelerate_sagemaker_examples/README.md:4]:
```suggestion
1. Set up the accelerate config by running `accelerate config --config_file accelerate_config.yaml` and answer the SageMaker questions.
```
(Set up is already at the beginning)

File: sagemaker/22_accelerate_sagemaker_examples/README.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+3. One can specify a custom docker image instead of Official 🤗 DLCs through the accelerate config questionnaire. When this isn't provided, the latest Official 🤗 DLC will be used.
+
+4. Support for input channels pointing to S3 data locations via TSV file, e.g., below are the contents of sagemaker_inputs.tsv whose location is given as part of accelerate config setup.
+```tsv
+channel_name	data_location
+train	s3://sagemaker-sample/samples/datasets/imdb/train
+test	s3://sagemaker-sample/samples/datasets/imdb/test
+```
+
+6. Support for SageMaker metrics logging via TSV file, e.g., below are the contents of the sagemaker_metrics_definition.tsv whose location is given as part of accelerate config setup.
```
Comment (sgugger) [inline sagemaker/22_accelerate_sagemaker_examples/README.md:33]: There is no 5 ;-)

**Review by sgugger — APPROVED** [review]
Oh very nice, thanks a lot for adding this!

File: sagemaker/22_accelerate_sagemaker_examples/README.md (hunk 3 above)
Comment (pacman100) [reply sagemaker/22_accelerate_sagemaker_examples/README.md]: resolved.

**Pull request merged**